tests/components/config/test_automation.py

```diff
@@ -25,10 +25,10 @@ def stub_blueprint_populate_autouse(stub_blueprint_populate: None) -> None:
 
 @pytest.fixture
 async def setup_automation(
-    hass,
+    hass: HomeAssistant,
     automation_config,
-    stub_blueprint_populate,
-):
+    stub_blueprint_populate: None,
```

Comment:
Instead of typing this None, why not pull it out into the decorator? As in, this doesn't have to be an argument at all?

Reply:
`pytest.mark.usefixtures` is not allowed on `pytest.fixtures`:
https://docs.astral.sh/ruff/rules/pytest-erroneous-use-fixtures-on-fixture/

Reply:
Right... I should have picked a different spot to comment on...